test/test_libs.py

```diff
@@ -1590,7 +1590,7 @@ def make_vmas():
             [n_workers, list(env.num_envs)[0], n_rollout_samples]
         )
 
-    @pytest.mark.parametrize("num_envs", [1, 10])
+    @pytest.mark.parametrize("num_envs", [1, 2])
```

Comment:
vmas envs are quite cheap, maybe I would keep 10 in the vecotrized dimension to make sure we do not fall in a particular case

Reply:
@matteobettini
sorry I missed that
they're not very cheap actually, compared to other libs running the vmas tests take a considerable amount of time and torchrl is already a very intensive CI for a small library. I'd rather keep small numbers unless we have a strong reason to have large ones